### DISCUSSION TITLE
version 1.4.2

### DISCUSSION THREAD

**Discussion by alex19EP** [RHVoice/RHVoice#395] [Show and tell]
<hr /><em>This discussion was created from the release <a href='https://github.com/RHVoice/RHVoice/releases/tag/1.4.2'>version 1.4.2</a>.</em>

**Comment by zstanecic** [comment]
What are we going to discuss here?

 

From: Alexander Epaneshnikov ***@***.***> 
Sent: Friday, September 10, 2021 11:27 PM
To: RHVoice/RHVoice ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [RHVoice/RHVoice] version 1.4.2 (Discussion #395)

 

  _____  

This discussion was created from the release version 1.4.2 <https://github.com/RHVoice/RHVoice/releases/tag/1.4.2> . 

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub <https://github.com/RHVoice/RHVoice/discussions/395> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/ACVCDE34GBZY5H4LOSRKKC3UBJZZ5ANCNFSM5D2EOQAQ> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .   <https://github.com/notifications/beacon/ACVCDE354E7QXAFRCFVQ4Z3UBJZZ5A5CNFSM5D2EOQA2YY3PNVWWK3TUL52HS4DFVJCGS43DOVZXG2LPN2VGG33NNVSW45C7NFSM4ABWNGMQ.gif>

**Reply by alex19EP** [reply]
absolutely anything. it's just an auto-generated topic from the release.

**Comment by CyberTailor** [comment]
Could you upload version 1.6.0? Thanks.
